src/registrar/templates/home.html

```diff
@@ -33,26 +33,43 @@ <h2>Domains</h2>
       </thead>
       <tbody>
         {% for domain in domains %}
-        {% comment %} ticket 796
-        {% if domain.application_status == "approved" or (domain.application does not exist)  %} {% endcomment %}
         <tr>
           <th th scope="row" role="rowheader" data-label="Domain name">
             {{ domain.name }}
           </th>
           <td data-sort-value="{{ domain.created_time|date:"U" }}" data-label="Date created">{{ domain.created_time|date }}</td>
-          <td data-label="Status">{{ domain.application_status|title }}</td>
+          <td data-label="Status">
+            {% if domain.state == "unknown" or domain.state == 'dns needed'%}
```

Comment:
```suggestion
            {% if domain.state == domain.State.UNKNOWN or domain.state == domain.State.DNS_NEEDED%}
```
(Optional)

Reply:
The context does not exist on this template (it does on domain_detail)

Reply:
Extreme nitpick: `"dns needed"` just to match the str format :)

Reply:
Fair enough